src/components/Footer/index.tsx

```diff
@@ -1,5 +1,5 @@
 'use client'
-import { Logo } from '@/src/assets/svg'
+import Logo from '../Logo/Logo'
```

Comment:
We will move all SVG to one folder. `src/styles` folder.

Reply:
@rmc3408 do you want move one asset like SVG to a folder named styles which mean how the component looks like.